docs/source/config/settings.rst

```diff
@@ -388,6 +388,14 @@ Development
 
     Default: ``'https://hub.getindico.io'``
 
+.. data:: DISABLE_CELERY_CHECK
+
+    Disables the warning about Celery not running / being outdated. By default
+    it is disabled when :data:`DEBUG` is enabled, but this setting can be used
+    to explicitly enable/disable the warning regardless of debug mode.
+
+    Default: ``None``
```

Comment:
The default is None?

Reply:
See the text above - if unset, it depends on the `DEBUG` option.